### DISCUSSION TITLE
Kit claim full inventory check

### DISCUSSION THREAD

**Discussion by rmalca** [Zrips/CMI#6166] [Enhancement]
Please add an option so that when claiming a kit and the inventory is full, the items do not fall to the ground, instead show a message saying to free up space in inventory.

**Comment by mrfloris** [comment]
I believe the latest CMI version now has this. 
https://www.spigotmc.org/resources/cmi-298-commands-insane-kits-portals-essentials-economy-mysql-sqlite-much-more.3742/update?update=424296

```
Fix for items from kits disappearing in case you have kits set to not drop items if inventory is full. Now we will inform the player about not enough space to get this kit
```
